src/hydra/hydraClient.js

```diff
@@ -341,25 +342,32 @@ export default ({entrypoint, resources = []}, httpClient = fetchHydra) => {
   const fetchApi = (type, resource, params) => {
     // Hydra doesn't handle MANY requests, so we fallback to calling the ONE request n times instead
     switch (type) {
+      case "INTROSPECT":
+        if (apiSchema) return Promise.resolve(apiSchema);
+        return apiDocumentationParser(entrypoint).then(({ api }) => {
```

Comment:
The recursive retrieving of the schema (for all the resources) is missing.

https://github.com/api-platform/admin/blob/4004cc6152ebdf14455d3f8f2511a5ce57823f09/src/hydra/HydraAdmin.js#L35-L53